app/src/main/java/com/github/se/studybuddies/ui/map/MapScreen.kt

```diff
@@ -123,46 +152,140 @@ fun MapScreen(
                       snippet = "This is your current location")
                 }
               }
+              friendsData.value.forEach { friend ->
+                if (friend.location != "offline" && friend.location != "") {
```

Comment:
For example here, instead of writing offline as a hard-coded string yourself, you'd just check `friend.location != LocationState.OFFLINE`

If you think it's a good idea, you can just put the enum in User.kt